app/src/main/java/com/nytimes/storedemo/ui/MainView.java

```diff
@@ -37,17 +42,19 @@ public MainView(Context context, AttributeSet attrs, int defStyleAttr) {
     protected void onFinishInflate() {
         super.onFinishInflate();
         presenter.bind(this);
+        articleRecyclerView = (ArticleRecyclerView) findViewById(R.id.articleRecyclerView);
 
         FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
 
         RxView.clicks(fab)
                 .flatMap(aVoid -> presenter.getArticles())
                 .subscribe(articles -> {
-            displayArticles();
+            displayArticles(articles);
         });
     }
 
-    private void displayArticles() {
+    private void displayArticles(List<Article> articles) {
+        articleRecyclerView.setArticles(articles);
```

Comment:
Method is unneeded. Instead make setArticles return an action and plug that directly into the subscribe